cmd/zoekt-archive-index/e2e_rank_test.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sourcegraph/zoekt"
+	"github.com/sourcegraph/zoekt/build"
+	"github.com/sourcegraph/zoekt/query"
+	"github.com/sourcegraph/zoekt/shards"
+)
+
+var update = flag.Bool("update", false, "update golden file")
+
+func TestRanking(t *testing.T) {
```

Comment:
This was done purely for convenience since this contains logic for fetching archives from github. WDYT about refactoring this out into a shared package and using it there? Then the test will be in a more reasonable place?

Reply:
I like that idea, fetching and indexing GitHub archives feels generally useful.

Reply:
Finally did this suggestion https://github.com/sourcegraph/zoekt/pull/712